sktime/benchmarking/evaluation.py

```diff
@@ -111,7 +111,7 @@ def evaluate(self, metric, train_or_test="test", cv_fold="all"):
         # aggregate over cv folds
         metrics_by_strategy_dataset = (
             self._metrics.groupby(["dataset", "strategy"], as_index=False)
-            .agg(np.mean)
+            .agg("mean")
```

Comment:
Question: are these different?

Reply:
yes, `pandas` only likes string arguments since recently, this is what causes the deprecation warning

Reply:
I think it's not a deprecation, they'll continue to support both (guessing from `FutureWarning`).

```python
>>> import pandas
>>> df = pandas.DataFrame({"a": [1, 2, 3, 4], "b": [5, 6, 7, 8]})
>>> import numpy
>>> df.groupby("a").agg(numpy.mean)
<stdin>:1: FutureWarning: The provided callable <function mean at 0x7fa9e0793640> is currently using DataFrameGroupBy.mean. In a future version of pandas, the provided callable will be used directly. To keep current behavior pass the string "mean" instead.
     b
a
1  5.0
2  6.0
3  7.0
4  8.0
>>>
```